runtime/lua/vim/filetype.lua

```diff
@@ -1007,6 +1036,156 @@ local extension = {
       return 'text'
     end
   end,
+  cmd = function(path, bufnr)
+    if getline(bufnr, 1):find('^/%*') then
+      return 'rexx'
+    end
+    return 'dosbatch'
```

Comment:
Add an `else` here like for the `xpm` function for consistency (or change `xpm` instead).

Reply:
Personally I prefer

```lua
if cond then
  return truthy_val
end

return falsey_val
```

to

```lua
if cond then
  return truthy_val
else
  return falsey_val
end
```

Especially when the `else` branch is multiple lines (it saves an unnecessary indent). However, I won't insist on it if it adds a lot of churn.

Reply:
I personally don't care enough not to agree with @gpanders, but I do like consistency :)